includes/db.inc

```diff
@@ -35,9 +35,14 @@ function islandora_usage_stats_pid_id($pid) {
  */
 function islandora_usage_stats_pid_to_db($pid) {
   $object = islandora_object_load($pid);
-  return db_insert('islandora_usage_stats_objects')
-    ->fields(array('pid' => $pid, 'label' => $object->label))
-    ->execute();
+  if ($object) {
+    return db_insert('islandora_usage_stats_objects')
+      ->fields(array('pid' => $pid, 'label' => $object->label))
```

Comment:
Really, the object is just loaded here to get the label... Instead of excepting out, could just dump `NULL` or some placeholder for the label, yeah?

Something like:
```php
return db_insert('islandora_usage_stats_objects')
  ->fields(array(
    'pid' => $pid,
    'label' => ($object ? $object->label : NULL),
    ))
    ->execute();
```